stdlib/concurrent/futures/process.pyi

```diff
@@ -19,8 +19,9 @@ _global_shutdown: bool
 
 class _ThreadWakeup:
     _closed: bool
-    _reader: Connection
-    _writer: Connection
+    # Any: Unused send and recv methods
+    _reader: Connection[Any, Any]
+    _writer: Connection[Any, Any]
```

Comment:
Maybe these should be `Connection[object, object]` or `Connection[Unused, Unused]`? Otherwise we might get complaints from the (very small) group of people who use mypy's `--disallow-any-expr` optional lint: https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-disallow-any-expr

Reply:
For the first, `_SendT`, contravariant, used for params argument sure.
Idk about the second, `_RecvT`, covariant, used for return type.

```python
reader1: Connection[int, object] = _ThreadWakeup()._reader  # OK
reader2: Connection[object, int] = _ThreadWakeup()._reader  # "Connection[object, object]" cannot be assigned to declared type "Connection[object, int]"

class MyThreadWakeup(_ThreadWakeup):
    _reader: Connection[int, int]  # OK
```

Arguably this is also a "private" name and I fail to come up with a real-world example

Reply:
One option is to use the stricter non-Any types for now, and see if anybody complains about false positives... But I'm also fine with the PR being merged as-is; I don't have a strong opinion here :-)